frontend/src/components/application/attachments/DeleteAttachmentModal.tsx

```diff
@@ -0,0 +1,81 @@
+"use client";
+
+import { useTranslations } from "next-intl";
+import { RefObject } from "react";
+import {
+  Button,
+  ButtonGroup,
+  ModalFooter,
+  ModalRef,
+  ModalToggleButton,
+} from "@trussworks/react-uswds";
+
+import { SimplerModal } from "src/components/SimplerModal";
+import Spinner from "src/components/Spinner";
+
+interface Props {
+  buttonCtaText: string;
```

Comment:
since this is already specific to deleting attachments do we need these values to be flexible? can we cut down on the props by just getting these strings within the component? looks like we could cut this down to 4 props